src/js/views/renderCharacter.js

```diff
@@ -25,8 +25,9 @@ export const characterHTML = `
     </div>
 `;
 
-export default function renderCharacters() {
-  for (let i = 0; i < 4; i += 1) {
+export default function renderCharacters(arr) {
+  console.log(arr);
+  for (let i = 0; i < arr.length; i += 1) {
```

Comment:
Kind of a nit-picky thing, but maybe it would be better to use a `for of` or `forEach` loop here, instead of a normal for loop.
```
arr.forEach((character) => {
    .....
});
```
^ In this case if we wanted to access a value from this array in the future, we could do `character.name` instead of `arr[i].name`.
Although this isn't a big deal at all for this case in particular.

Reply:
Not nit-picky at all! I appreciate you pointing this out. I do think a forEach looks cleaner, and I'm going to work on this change. I'll let you know when I make the commit so you can check it out. Thanks Bennett! :)